package.json

```diff
@@ -16,25 +16,33 @@
     "precompile-svgs": "node utils/precompileSvgs/index.js && yarn run pregenerate",
     "precompile-custom-svgs": "node utils/precompileSvgs/index.js --custom && yarn run pregenerate",
     "_lint-watch-fix": "yarn lint -w -m",
-    "test": "jest --config=jest.conf/index.js",
+    "test": "yarn test:unit && yarn test:visual",
+    "test:unit": "jest --config=jest.conf/index.js",
+    "test:visual": "JEST_PUPPETEER_CONFIG=./jest-puppeteer.config.js node start-server-and-test.js",
     "_api-watch": "chokidar \"**/lib/**\" -c \"node utils/extractApi.js\""
   },
   "files": [
     "lib"
   ],
   "dependencies": {
+    "@percy/cli": "^1.28.7",
+    "@percy/puppeteer": "^2.0.2",
     "@vue/composition-api": "1.7.2",
     "aphrodite": "https://github.com/learningequality/aphrodite/",
     "autosize": "3.0.21",
     "css-element-queries": "1.2.0",
     "date-fns": "1.30.1",
+    "fkill": "^9.0.0",
     "frame-throttle": "3.0.0",
     "fuzzysearch": "1.0.3",
+    "jest-puppeteer": "^10.0.1",
     "lodash": "4.17.21",
     "popper.js": "1.16.1",
+    "puppeteer": "^22.11.0",
     "purecss": "2.2.0",
     "tippy.js": "4.2.1",
     "vue-intl": "3.1.0",
+    "wait-on": "^7.2.0",
```

Comment:
I dont see this `wait-on` lib used anywhere, probably it was someething you were testing and then removed? :sweat_smile:

Reply:
Oh I had used this while configuring jest-puppeteer but removed that part later on. Will remove this dependency.